error with HTTPClient

Hello. I am trying to compile this project and I have some issues with HttpClient.

`$pio run`
 gives me an error : 
`src/wifiMgr.cpp:749:24: fatal error: HttpClient.h: No such file or directory`

So I installed lib #66 (HttpClient v.2.2.0)

Then, I have other errors :
`src/wifiMgr.cpp: In function 'void handleRadioConfig(AsyncWebServerRequest*)':
src/wifiMgr.cpp:442:49: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
         saveToSPIFFS(WIFI_SETUP_FILE, p->value());
                                                 ^
src/wifiMgr.cpp: In function 'void getCoverBMID(char*, char*)':
src/wifiMgr.cpp:757:5: error: 'HTTPClient' was not declared in this scope
     HTTPClient http;
     ^
src/wifiMgr.cpp:768:5: error: 'http' was not declared in this scope
     http.begin(buf);
     ^
src/wifiMgr.cpp: In function 'void getCoverJPG(char*)':
src/wifiMgr.cpp:796:5: error: 'HTTPClient' was not declared in this scope
     HTTPClient http;
     ^
src/wifiMgr.cpp:806:5: error: 'http' was not declared in this scope
     http.begin(buf);
     ^
` 
[error log.txt](https://github.com/ThomasH-W/ESP32-Radio-TTGO/files/7723282/error.log.txt)

I attach the full buid log

--- a/src/wifiMgr.cpp
+++ b/src/wifiMgr.cpp
@@ -746,7 +746,7 @@ char *urlencode(char *dst, char *src)
     return dst;
 }
 
-#include "HttpClient.h"
+#include "HTTPClient.h"
 String serverName = "http://musicbrainz.org/ws/2/release/?fmt=json&limit=1&query=release:";
 //                   http://musicbrainz.org/ws/2/release/?limit=1&query=release:Welshly%20Arms%20%20/%20Legendary
 //                   http://musicbrainz.org/ws/2/release/?fmt=json%26limit=1%26query=artist:Michael%20Patrick%20Kelly%20AND%20release:Beautiful%20Madness%20

